Raven.Tests.Helpers/RavenTestBase.cs

```diff
@@ -865,5 +962,65 @@ protected static void DeployNorthwind(DocumentStore store, string databaseName =
 			var request = requestFactory.CreateHttpJsonRequest(new CreateHttpJsonRequestParams(null, url + "/studio-tasks/createSampleData", "POST", store.DatabaseCommands.PrimaryCredentials, store.Conventions));
 			request.ExecuteRequest();
 		}
+
+        private static void StoreSeedData(IEnumerable<IEnumerable> seedData,
```

Comment:
Damn it. Just added 2 more methods with a different (last) overload and the calling code doesn't know which one to use (if no seed data was provided).

So I'll just leave the PR as it is then. Drats! was hoping to be tricky :)